torch/_torch_docs.py

```diff
@@ -3152,7 +3152,24 @@ def merge_dicts(*dicts):
            r"""
 flatten(input, start_dim=0, end_dim=-1) -> Tensor
 
-Flattens a contiguous range of dims in a tensor.
+By default, flattens :attr:`input` into a one-dimensional tensor with the same number of elements.
+If :attr:`start_dim` or :attr:`end_dim` are passed, only dimensions included in the range are
```

Comment:
"included in the range" is vague because the range is not specified. This could say something like:

"are passed, then end_dim must be a dimension after start_dim, and only the dimensions starting at start_dim and ending at end_dim are flattened." ?